src/Helpers/Tokens/TokenVerifier.php

```diff
@@ -136,12 +132,13 @@ public function verify(string $token, array $options = []) : array
         $now    = $options['time'] ?? time();
         $leeway = $options['leeway'] ?? $this->leeway;
 
-        $tokenExp = $verifiedToken->getClaim('exp', false);
-        if (! $tokenExp || ! is_int($tokenExp)) {
+        $tokenExp = $verifiedToken->claims()->get('exp', false);
+
+        if (! $tokenExp instanceof \DateTimeImmutable) {
```

Comment:
After parsing the token 'exp' and 'iat' will be the instance of DateTimeImmutable